CONTRIBUTING.rst

```diff
@@ -151,3 +151,20 @@ folders with pascal code can be found:
 We refer to the documentation of `Pascal <https://www.freepascal.org/docs.html>`_
 and the `Lazarus IDE <https://www.lazarus-ide.org/>`_ to get started with
 developping.
+
+Making a new release
+--------------------
+
+When a new release will be published on GitHub make sure following steps are taken:
```

Comment:
question: is a the generation of a release not automatically done by using tags? If so, part of these instructions will need to be added (https://watem-sedem.github.io/pywatemsedem/contributing.html#package-release)